index.html

```diff
@@ -1,172 +1,202 @@
-                    <p class="picture-text">Kjellerbandet</p>
-                </a>
-                <a class="picture-container" href="http://www.leisuresuitlovers.com/">
-                    <img src="assets/img/lsl.jpg" alt="">
-                    <p class="picture-text">Leisure Suit Lovers</p>
-                </a>
-                <a class="picture-container" href="http://smoller.no">
-                    <img src="assets/img/smoller.jpg" alt="">
-                    <p class="picture-text">S. Møller Storband</p>
-                </a>
-                <a class="picture-container" href="http://snaustrinda.samfundet.no">
-                    <img src="assets/img/snaustrinda.jpg" alt="">
-                    <p class="picture-text">Snaustrinda Spelemannslag</p>
-                </a>
-                <a class="picture-container" href="http://salong.samfundet.no">
-                    <img src="assets/img/salongen.jpg" alt="">
-                    <p class="picture-text">Studentersamfundets Salongorkester</p>
-                </a>
-                <a class="picture-container" href="http://symforch.no">
-                    <img src="assets/img/symforch.jpg" alt="">
-                    <p class="picture-text">Studentersamfundets Symfoniorkester</p>
-                </a>
-            </div>
-        </div>
-        <div class="content-container">
-            <div class="heading">
-                <h3>Vil du spille?</h3>
-            </div>
-            <div class="two-column-container">
-                <div class="image-container">
-                    <img src="assets/img/lsl.jpg" alt="Foto av Leisure Suit Lovers på scenen">
-                    <p class="image-text">LSL-konsert for en full Storsal | 2019 | <a href="https://foto.samfundet.no/">foto.samfundet.no</a></p>
-                </div>
-                <div class="text-container">
-                    <p>Er du student i Trondheim og ønsker å utfolde deg musikalsk bør Musikerlåfte være et interessant sted for likesinnede. De fleste radisjonelle musikksjangrene er representert blant vår undergrupperinger som alle har egne hjemmesider du kan klikke deg til via bildene over.</p>
-
-                    <p>Ønsker du å spille sammen med oss kan vi garantere et høyt musikalsk nivå og et fantastisk sosialt miljø. Du vil oppleve det beste ved studenttilværelsen i Trondheim, med gjengmedlemskap på Studentersamfundet og alle goder det fører med seg, samt muligheten til å fortsette med det du liker aller best nemlig å spille.</p>
-
-                    <p>Se <a href="views/recruitment.html">opptaksiden</a> for å hvilke instrumenter vi er på utkikk etter akkurat nå!</p>
-                </div>
-            </div>
+      </section>
+      <section>
+        <h2>Vil du spille?</h2>
+        <div class="want-to-play-row">
+          <div class="image-container">
+            <img src="assets/img/lsl.jpg" alt="LSL-konsert i storsalen" />
```

Comment:
"Foto av Leisure Suit Lovers på scenen" endret jeg til for noen uker siden. Storsalen står allerede i brødtekst under. Fint å ha med "Foto av", er det en tanke bak hvorfor dette ble endret?

Reply:
Mente alt-teksten da...